samples/customer-support-agent/agent/utils.py

```
@@ -0,0 +1,41 @@
+from langchain_core.runnables import RunnableLambda
+
+from langgraph.prebuilt import ToolNode
+
+
+def handle_tool_error(state) -> dict:
+    error = state.get("error")
+    tool_calls = state["messages"][-1].tool_calls
+    return {
+        "messages": [
+            ToolMessage(
+                content=f"Error: {repr(error)}\n please fix your mistakes.",
```

Ruff (0.14.6)

Use explicit conversion flag

Replace with conversion flag

(RUF010)